src/commands/emote.js

```diff
@@ -28,7 +31,43 @@ module.exports = {
 					logger.info(`Emote attached: ${response}`);
 				}
 			} catch (error) {
-				logger.error(`Emote failed to send: ${error}`);
+				sendErrorMessage(`Emote failed to send: ${error}`, message.channel);
+			}
+
+			try {
+				await message.delete();
+			} catch (error) {
+				logger.error(`Error deleting the message: ${error}`);
+			}
+		}
+	},
+	mEmote : {
+		name : 'm-emote',
+		args : true,
+		description : 'Sends a bigger version of an emote',
+		execute: async(message, data) => {
+			logger.info('multi-emote command...');
+			const emojis = data.filter(possibleEmoji => {
+				return (possibleEmoji[0] === '<' && possibleEmoji.slice(-1) === '>');
+			}).map(emoji => {
+				let fileType;
+				if (emoji[1] === 'a') {
+					fileType = '.gif';
+					emoji = emoji.replace('a', '');
+				} else {
+					fileType = '.png';
+				}
+				const emojiId = parseEmojiText(emoji);
+				return `https://cdn.discordapp.com/emojis/${emojiId + fileType}`;
+			}).join(' ');
+
+			try {
+				let response = await message.channel.send(emojis);
+				if (response) {
+					logger.info(`Emote attached: ${response}`);
```

Comment:
When is it a good time to log things? I know we should do it on errors, connecting to third party applications, or things that can fail in general. What makes this instance a good time to log something good?

Reply:
You want to log things when you need to handle tracing a transaction. For example, we want to know when an Emote gets attached and deleted or modified in any way. Thus, if an issue arises in the future where we get an expected result. We can trace through the logs to see any and all modifications that occurred to that emote.